src/events/data.json

```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "title": "The Future of Blockchain Solution Development",
+    "start": "Weds, May 22, 2019, 3:00 PM EST",
```

Comment:
we're in EDT not EST

Reply:
I've started omitting the daylight/standard designation when spelling out times for this reason. Perhaps change them to "ET"?